RELEASE-NOTES.md

```diff
@@ -67,6 +72,7 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 - Slam: `IndirectionPolicy::data()` now returns a reference to the underlying buffer
   Rebinding an indirection to a new buffer is now achieved through `IndirectionPolicy::ptr()`, which
   returns a mutable pointer to the buffer.
+- `Shaper::applyTransforms()` is no longer a public method.
```

Comment:
👍 

Thanks for adding this. 
It might be good to highlight that this is in quest, e.g.:
```suggestion
- Quest: `Shaper::applyTransforms()` is no longer a public method.
```

Reply:
I added "Quest:" to my changes in the release notes.